### DISCUSSION TITLE
Weekly community meeting!

### DISCUSSION THREAD

**Discussion by petemoore** [taskcluster/taskcluster#5036] [Announcements]
Every Friday at 17:00 CET/CEST, we welcome you to a Taskcluster Community meeting.

Feel free to drop by our Zoom call, and get involved in our activities.

* [Google calendar](https://calendar.google.com/calendar/embed?src=c_q37pqgifbrmgngdghp0fej2kts%40group.calendar.google.com&ctz=Europe%2FBerlin)
* [Meeting notes](https://docs.google.com/document/d/1dCQScxwH04VInIntxesYVQ8te1IvLTZXWYrHNgi1Irc/edit?usp=sharing)
* [Zoom meeting](https://mozilla.zoom.us/j/96290916993)

**Comment by petemoore** [comment]
Thanks everyone for attending the first meeting last week, great to see you all, and to get the ball rolling. :-)

**Comment by petemoore** [comment]
Note, meeting notes are here: https://docs.google.com/document/d/1dCQScxwH04VInIntxesYVQ8te1IvLTZXWYrHNgi1Irc/edit?usp=sharing

**Comment by petemoore** [comment]
<img width="1840" alt="Screenshot 2024-09-06 at 17 38 08" src="https://github.com/user-attachments/assets/3cbc35a3-3eed-4d77-8fa4-7bef524dd520">
An example Taskcluster Community meeting on 6 September 2024.

Permission was received from all attendees to share the screenshot here.

**Comment by nitishagar** [comment]
@petemoore I'm not able to join the call it gives the following msg: 
> The host requires authentication on the commercial Zoom platform to join this meeting. Please [sign in](javascript:;) with a commercial Zoom account to join.

**Reply by petemoore** [reply]
@nitishagar Ah no, I'm sorry that happened! The meeting was cancelled, and this may be the reason. Normally anybody can join, but maybe the host needs to be connected. Apologies about that. Note we also have a matrix channel: https://matrix.to/#/#taskcluster:mozilla.org where usually we would communicate if the meeting will be cancelled and we try to update the meeting notes ahead of time, but this does not always happen. Hopefully you can join us a different time, and thanks for trying this time!
